homeassistant/components/upnp/strings.json

```
@@ -15,9 +15,8 @@
       }
     },
     "abort": {
-      "already_configured": "UPnP/IGD is already configured",
-      "no_devices_discovered": "No UPnP/IGDs discovered",
-      "no_devices_found": "No UPnP/IGD devices found on the network.",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
```

Would the abort be triggerd for a duplicate intergration or duplicate router ip addres.
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
```
https://github.com/home-assistant/core/blob/b7f78c7fbb1bf3ec2a2b343103df51f93f055cf1/homeassistant/components/upnp/config_flow.py#L108-L111

---

Looks like line #108 says device 👍